atlas/src/shared/components/VideoPreview/VideoPreview.styles.tsx

```diff
@@ -13,6 +13,9 @@ type ChannelProps = {
 
 export const CoverImage = styled.img<CoverImageProps>`
   display: block;
+  position: absolute;
```

Comment:
I don't think the image should be `absolute` here. What's stopping us from having it be a part of the layout?

Reply:
I'm setting the image to be `absolute` because I'm using a "trick" to preserve the aspect ratio of the images.
I absolutely position each image in a `div` which has a width of 100% and a `padding-top` of `100% * aspectRatio`,  so the absolute positioning is required in this case. 
I'm happy to change this if there is a better way though.